### DISCUSSION TITLE
can we put widgets in amazfit band 7 . i want a custom widgets like water reminder widgets.

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#64] [Q&A]
as per the title if we can please give me the code .

**Comment by MyoungXUE** [comment]
Hi, the widgets on the Zepp OS 1.0 documentation are available on band 7 devices.

Can you provide a picture of the water reminder widgets you are referring to, so I can provide you with a code snippet for your reference?
